src/Nethermind/Nethermind.Evm/EOF/EvmObjectFormat.cs

```diff
@@ -280,37 +284,103 @@ public bool ValidateBody(ReadOnlySpan<byte> container, in EofHeader header)
         bool ValidateInstructions(ReadOnlySpan<byte> code, in EofHeader header)
         {
             int pos;
-            for (pos = 0; pos < code.Length; pos++)
+            Span<byte> codeBitmap = stackalloc byte[(code.Length / 8) + 1 + 4];
```

Comment:
I think stackalloc here is not a good idea as it can be XX kB!
I think better idea would be to use ArrayPool<byte>